karma.conf.js

```diff
@@ -82,7 +82,7 @@ module.exports = function(config) {
 
     // start these browsers
     // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
-    browsers: ['PhantomJS'],
+    browsers: ['ChromeHeadless'], //['PhantomJS'],
```

Comment:
Are we definitively switching to ChromeHeadless? If so, you can remove the PhantomJS part